.github/workflows/go.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.16
+        go-version: 1.18
```

Comment:
1.19 according to https://github.com/redhat-developer/observability-operator/blob/16c9ed525e38d7f4ad2e7d5ee8f80f0ec2f4bda7/go.mod#L3?

Reply:
Thanks @machi1990 I'll update that